packages/engine/src/avatar/components/TransparencyDitheringComponent.ts

```diff
@@ -54,15 +54,15 @@ export const TransparencyDitheringComponent = defineComponent({
     return {
       ditheringDistance: 0.35,
       ditheringExponent: 5,
-      position: new Vector3()
+      center: new Vector3()
     }
   },
 
   onSet: (entity, component, json) => {
     if (!json) return
     if (matches.number.test(json.ditheringDistance)) component.ditheringDistance.set(json.ditheringDistance)
     if (matches.number.test(json.ditheringExponent)) component.ditheringExponent.set(json.ditheringExponent)
-    if (matchesVector3.test(json.position)) component.position.set(json.position)
+    if (matchesVector3.test(json.center)) component.center.value.copy(json.center)
```

Comment:
for proper reactivity, we need to use `component.center.set(new Vector3().copy(json.center))`